src/SqlProviderBase.ts

```diff
@@ -25,6 +25,13 @@ function getIndexIdentifier(storeSchema: NoSqlProvider.StoreSchema, index: NoSql
     return storeSchema.name + '_' + index.name;
 }
 
+// Certain indexes use a separate table for pivot:
+// * Multientry indexes
+// * Full-text indexes that support FTS3
+function indexUsesSeparateTable(indexSchema: NoSqlProvider.IndexSchema, supportsFTS3: boolean): boolean {
```

Comment:
The supportsFTS3 value is always passed this._supportsFTS3, have you considered making this a private function inside the class and using that value?